dumper/app/src/main/resources/oracle-stats/dba/awr/source-conn-latest.sql

```diff
@@ -0,0 +1,41 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  History.dbid "DbId",
```

Comment:
Can we add `ConId` column to `source-conn-latest`? I can see it is available in `hist_active_sess_history` view. This allows to add filtering on database name in the report.

This can be added in the separate PR ofc.

Reply:
Sure, I added `ConId` like in other SQLs